zebrunner.sh

```diff
@@ -1188,7 +1196,7 @@ case "$1" in
         if [ -z $2 ]; then
           start
         else
-         start-device $2
+          start-device $2
         fi
```

Comment:
Why new space symbol.

Reply:
Probably I didn't want to add that on purpose. I'll revert the space.